handlers/routes.py

```diff
@@ -272,7 +272,7 @@ def authenticate_json(json_data):
         return False
 
 
-def render_send_by_link_id(link_id, email, name):
+def render_send_by_link_id(link_id, email, name, email_body_html="", email_body_text=""):
     """Download and render and then sign a document from google and send it by email"""
```

Comment:
Creo suena mejor así
`"""Download and render a document from google then sign and send it by email"""`

Reply:
si tienes razon :)